Scripts/run-tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -o pipefail
+exitCode=0
+
+function checkStatus {
+    if [[ $exitCode == 0 && $1 != 0 ]]; then
+        exitCode=$1
+    fi
+}
+
+function uploadCodecov {
+    bash <(curl -s https://codecov.io/bash)
+}
+
+xcodebuild build -scheme "RestKit" -sdk "$IOS_SDK" -destination "$IOS10_DESTINATION" | xcpretty
```

Comment:
I think we should call the destination variable `IOS_DESTINATION`.  The current name seems to imply that it should only be set to an iOS 10 value, but we might want to use iOS 9 -- or soon iOS 11.